ethereum/jsonrpc/src/test/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/privacy/methods/priv/PrivCreatePrivacyGroupTest.java

```diff
@@ -41,27 +41,30 @@
         "A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=",
         "Ko2bVqD+nNlNYL5EE7y3IdOnviftjiizpjRt+HTuFBs="
       };
-  private final String privacyGroupId =
-      "68/Cq0mVjB8FbXDLE1tbDRAvD/srluIok137uFOaClPU/dLFW34ovZebW+PTzy9wUawTXw==";
 
   @Test
   public void verifyCreatePrivacyGroup() throws Exception {
+    final String expected = "eBh8PRQYrZWK7T1uOIwtZWXFYaG1/MVhg+EBse8SxCU=";
     final PrivacyGroup privacyGroup =
-        new PrivacyGroup(privacyGroupId, PrivacyGroup.Type.PANTHEON, name, description, addresses);
+        new PrivacyGroup(expected, PrivacyGroup.Type.PANTHEON, name, description, addresses);
     when(enclave.createPrivacyGroup(any())).thenReturn(privacyGroup);
```

Comment:
agree on checking the type of object in the mock.

I don't see value in the refactor. Is there some principle I am missing here? I generally try to minimize the number method parameters as recommended by the Clean Code book :)

Reply:
Also, the request types are not Orion specific - they "should" also work with other Enclaves - but considering we only support Orion I think this can be pushed out to when we want to support other enclave technologies. We can then spec out interfaces more concretely. (potentially even lift this module out of pantheon into a web3j-enclaves project?!?!)